maps/petita.json

```
@@ -0,0 +1,8 @@
+{
+	"map_name": "Petita",
+	"map_path": "petita",
+	"map_file": "petita.dmm",
+	"station_name": "CRS Petita",
+	"station_name_ru": "��� ������",
```

```suggestion
	"station_name_ru": "ГИС Петита",
```

Не знаю, что тут случилось, но оно случилось.